ui/package.json

```diff
@@ -48,5 +48,11 @@
   },
   "jest": {
     "preset": "jest-puppeteer"
-  }
+  },
```

Comment:
I noticed that changes from this file tend to show up, but I always end up not committing them. What is this browserslist change? Should we commit it? Should we add it to the .gitignore?

Reply:
edit: nvm just saw your latest comment